client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-instructions/index.tsx

```diff
@@ -1,185 +1,151 @@
-									{
-										components: {
-											strong: <strong />,
-										},
-									}
-								) }
-								<div className="site-migration-instructions__migration-key">
-									<code className="site-migration-instructions__key">
-										{ hideSiteMigrationKey
-											? '*'.repeat( siteMigrationKey.length - 5 )
-											: siteMigrationKey }
-									</code>
-									<Button
-										onClick={ () => {
-											setHideSiteMigrationKey( ! hideSiteMigrationKey );
-										} }
-										className="site-migration-instructions__show-key-button components-button"
-									>
-										{ hideSiteMigrationKey ? translate( 'Show key' ) : translate( 'Hide key' ) }
-									</Button>
-									<ClipboardButton
-										text={ siteMigrationKey }
-										className="site-migration-instructions__copy-key-button is-secondary"
-										onCopy={ onCopy }
-									>
-										{ buttonTextCopy ? translate( 'Copied!' ) : translate( 'Copy key' ) }
-									</ClipboardButton>
-								</div>
-							</>
-						) }
-					</div>
-				</li>
 				<li>
 					{ translate(
-						'Go to the {{a}}Migrate Guru page on the source site{{/a}}, enter your email address, and click {{strong}}Migrate{{/strong}}.',
+						'Install and activate the {{a}}Migrate Guru plugin{{/a}} on your existing site.',
 						{
 							components: {
-								a: <a href={ sourceSiteUrl } target="_blank" rel="noreferrer" />,
-								strong: <strong />,
+								a: <a href={ getInstallationURL( fromUrl ) } target="_blank" rel="noreferrer" />,
 							},
 						}
 					) }
 				</li>
 				<li>
 					{ translate(
-						'You will see a screen showing multiple hosting providers. Select {{em}}Automattic{{/em}} as the destination host.',
+						'Go to the {{a}}Migrate Guru page on your source site{{/a}}, enter your email address, and click {{migrateButton /}}.',
 						{
 							components: {
-								em: <em />,
+								a: <a href={ getMigrateGuruPageURL( fromUrl ) } target="_blank" rel="noreferrer" />,
+								migrateButton: <DoNotTranslateIt value="Migrate" as="strong" />,
 							},
 						}
 					) }
 				</li>
 				<li>
 					{ translate(
-						'Find the {{em}}Migrate Guru Migration Key{{/em}} field, and paste the key you copied in step 2. Then click {{strong}}Migrate{{/strong}} to start your migration.',
+						'When asked to select a destination host, pick {{em}}WordPress.com{{/em}}.',
 						{
 							components: {
 								em: <em />,
-								strong: <strong />,
 							},
 						}
 					) }
 				</li>
-				<li>{ translate( 'Migrate Guru will send you an email when the migration finishes.' ) }</li>
+				{ isSuccess && migrationKey && (
+					<li>
+						{ translate(
+							'Copy and paste the migration key below in the {{ migrationKeyField /}} field and click {{migrateButton /}}.',
+							{
+								components: {
+									migrationKeyField: (
+										<DoNotTranslateIt value="Migrate Guru Migration key" as="em" />
```

Comment:
What's the reasoning behind adding a `DoNotTranslateIt` component? 

maybe adding the markup in the input string directly and passing the `Migrate Guru Migration key` | `Migrate` as an [arg](https://github.com/Automattic/wp-calypso/blob/trunk/packages/i18n-calypso/types/index.d.ts#L34) is enough?

Reply:
@andres-blanco 
The idea is to express the intent the value should not be translated, especially considering we don't have any kind of automated test testing translations.

My concern is we can easily remove the args value just because the intent is not clear and we don't have a test suite testing translations.

Reply:
ok, then my only comment is that it would be good to remove the markup from the `DoNotTranslateIt` element and keep it in the input string. That way is more readable, and also extendable (imagine if you want to do something like `<strong><em>STRING</em></strong>`).
But it's just a nitpick, not a blocker. Go with your gut :)

Reply:
Good point; I tested, and we can still do it via `{{strong}} {{migrationKeyField /}} {{/ strong }}.` but I still agree with you because it blocks the translators to freely move the formatting position. 
So, I update here https://github.com/Automattic/wp-calypso/pull/88813/commits/17eefe3a9baba824878040c44fb9501043d416b1